webpack/assets/javascripts/react_app/components/common/SubstringWrapper/SubstringWrapper.test.js

```diff
@@ -0,0 +1,36 @@
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import React from 'react';
+import SubstringWrapper from './';
+
+const setup = ({ substring, Element = 'b', text = 'this is a test' }) => (
+  <SubstringWrapper Element={Element} substring={substring}>
+    {text}
+  </SubstringWrapper>
+);
+
+describe('Search Input', () => {
+  it('should render with b tag', () => {
+    const wrapper = shallow(setup({ substring: 'test' }));
+
+    expect(toJson(wrapper)).toMatchSnapshot();
```

Comment:
A suggestion:
`const wrapper` seems like a redundancy. maybe doing it this way:

```
expect(toJson(shallow(setup({ substring: 'test' }))));
```

The same goes for the other 3 test cases below.

Reply:
:+1: 

In general, I would love to see the `testHelpers::testComponentSnapshotsWithFixtures` getting used more for this kind of tests
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBar.test.js#L21

Reply:
well I have mixed feeling with this.
it may be less readable:
```javascript
expect(toJson(shallow(setup({ substring: 'test', text: 'test - this is a test' })))).toMatchSnapshot();
````
therefore I prefer to extract the inner function.

Reply:
@amirfefer see how it done here https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBar.test.js#L21

I think it makes the tests much cleaner and based on configuration rather than code.

Reply:
The `testComponentSnapshotsWithFixtures` looks great, does it support children as well?
`SubstringWrapper` has a child (the wrapped text) - see how the `setup` is written.

Reply:
Yes! you can pass `{ children: 'some-text' }` or `{ children: <Component /> }`